app/src/main/java/com/andrew/tasky/presentation/agenda_item_detail/AgendaItemDetailFragment.kt

```diff
@@ -90,8 +104,27 @@ class AgendaItemDetailFragment: Fragment(R.layout.fragment_agenda_item_detail) {
                 }
             }
         }
+        collectLatestLifecycleFlow(viewModel.photos){
+            val adapter = PhotoItemAdapter(it, this)
+            binding.photosRecyclerView.adapter = adapter
+            binding.photosRecyclerView.layoutManager = LinearLayoutManager(
+                requireContext(), LinearLayoutManager.HORIZONTAL, false)
+            if (it.isEmpty()){
+                binding.addPhotoPlusSign.isVisible = true
+                binding.addPhotoTextView.isVisible = true
+                binding.photosTextView.isVisible = false
+                binding.photosRecyclerView.isVisible = false
```

Comment:
Instead of all these true/false values, just use the boolean from the if-statement:
`binding.addPhotoPlusSign.isVisible = it.isEmpty()`

That way, you don't need this if/else